jobs/integration/conftest.py

```diff
@@ -191,9 +199,9 @@ def juju_base(self, series):
         }
         return f"--base={mapping[series]}"
 
-    async def run(self, cmd: str, *args: str, stdin=None, _tee=False):
+    def exec(self, cmd: str, *args: str, stdin=None, _tee=False):
```

Comment:
`async subprocess` is actually really hard -- this method actually gets much simpler